lang/values.ts

```diff
@@ -0,0 +1,23 @@
+/** Returns true if the value is `null` or `undefined`. */
+export function isNil(value: unknown): value is null | undefined {
```

Comment:
Whoa! I have not seen this `value is null` type of return value. That's wild. I always learn multiple things from reviewing your TS code. 💯